<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>诗情密意 我要一个好记的密码 V1.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.png</normaloff>logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_recommend">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>86</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>再推荐一个</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_favorite">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>10</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>我有喜欢的诗句</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_poem">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>诗句</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_poeminput">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>421</width>
      <height>26</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_remark">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>备注</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_passwdoutput">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>421</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_trans">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>转换</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_passwd">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_remarkoutput">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>421</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
